R/tm_data_table.R

```diff
@@ -1,4 +1,4 @@
-#' Data Table Viewer Teal Module
+#' Data Table Viewer `teal` Module
```

Comment:
We maintain all titles and descriptions in sentence case, with titles not ending in a period.
```suggestion
#' Data table viewer `teal` module
```

Reply:
@kartikeyakirar this PR does not cover this change.
I think other titles in all other files are title-case.
Would you mind creating a separate issue to change titles into sentenace case for tmg?

Also, would you like to take a lead on the same issue on tmc https://github.com/insightsengineering/teal.modules.clinical/issues/1073 ?